.eslintrc.json

```diff
@@ -33,7 +33,13 @@
                             },
                             {
                                 "sourceTag": "scope:mini-rx-store-ng",
-                                "onlyDependOnLibsWithTags": ["scope:mini-rx-store"]
+                                "onlyDependOnLibsWithTags": ["scope:mini-rx-store", "scope:common"],
+                                "allowedExternalImports": [
```

Comment:
allowedExternalImports...

cool! maybe also for the other libs?

"rxjs/*", can be removed, because RxJS imports should import from "rxjs", also operator imports.

mini-rx-store could have "@angular/*" in bannedExternalImports because it is framework-agnostic.